cmd/kubeadm/app/discovery/token/token.go

```diff
@@ -171,6 +123,74 @@ func RetrieveValidatedConfigInfo(cfg *kubeadmapi.JoinConfiguration) (*clientcmda
 	return baseKubeConfig, nil
 }
 
+//getInsecureKubeconfigBytes is used to return insecureKubeconfigBytes from insecureBootstrapConfig and token
+func getInsecureKubeconfigBytes(insecureBootstrapConfig *clientcmdapi.Config, token *kubeadmapi.BootstrapTokenString) ([]byte, error) {
```

Comment:
Please elaborate comment to the new functions
This refactor is a great opportunity to better document how kubeadm validates cluster identity

Reply:
Thanks for your reminding, I will improve the comment.